spring-cloud-gcp-autoconfigure/src/test/java/org/springframework/cloud/gcp/autoconfigure/trace/StackdriverTraceAutoConfigurationTests.java

```diff
@@ -80,14 +84,20 @@ public void init() {
 
 		@Test
 		public void test() {
-			Span span = this.tracer.createSpan("Test Span");
-			this.tracer.close(span);
-
-			// Test that we are using 128bit Trace ID
-			Assert.assertNotEquals(0, span.getTraceIdHigh());
+			Span span = this.tracer.newTrace()
+					.kind(Span.Kind.CLIENT)
+					.name("test")
+					.start();
+			span.finish();
+			span.flush();
```

Comment:
I don't think you need to `flush` after `finish` but maybe I missed sth.

Reply:
ah right, i was testing to see if there were some differences. finish should be enough.